### PR TITLE
Style the page by editing how the elements are displayed

### PR DESCRIPTION
**Pull request by andreeanica16**

### CODE REVIEW

File: src/main/webapp/script.js [hunk 1]
```diff
@@ -736,6 +736,3 @@ function transformAgregatedDataforGeoChart(aggregatedData) {
     google.charts.setOnLoadCallback(drawRegionsMap(array, aggregatedData)); 
   });
 }
-
-module.exports = {getTotalCosts, getAverageCosts, getFailedJobs, getFailedJobsCost,
```
Comment (tblanshard) [inline src/main/webapp/script.js:740]: As I explained earlier, perhaps not very well, this needs to be in the code for testing. Whilst it doesn't do anything for the the web application itself (hence the error message that module doesn't exist), it allows the tests in the script.test.js file to run (they won't run without this line of code) - it's part of the testing framework.

**Comment by andreeanica16** [comment]
@iht @juliocc @tblanshard

File: src/main/webapp/script.js (hunk 1 above)
Comment (andreeanica16) [reply src/main/webapp/script.js:740]: Sorry for removing this line. I was trying to solve the error and forgot to add it back :D

File: src/main/webapp/script.js [hunk 2]
```diff
@@ -736,6 +736,3 @@ function transformAgregatedDataforGeoChart(aggregatedData) {
     google.charts.setOnLoadCallback(drawRegionsMap(array, aggregatedData)); 
   });
 }
```
Comment (tblanshard) [inline src/main/webapp/script.js:738]:
```suggestion
}
    module.exports = {getTotalCosts, getAverageCosts, getFailedJobs, getFailedJobsCost, getAveragevCPUCount, SSDVsHDDTimeComparison};
```  

Not entirely sure that the indentation was right as it's a little awkward to add but here you go :D

File: src/main/webapp/script.js (hunk 1 above)
Comment (tblanshard) [reply src/main/webapp/script.js:740]: No worries! I've added a comment above with the code back in so you should just be able to commit that (not entirely convinced the indentation is correct though)

**Comment by andreeanica16** [comment]
@iht @juliocc @tblanshard

**Review by tblanshard — APPROVED** [review]
LGTM :)

**Pull request merged**